CRM/Prospect/Upgrader/Steps/Step1008.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use CRM_Prospect_Service_ProspectingMenu as ProspectingMenu;
+
+/**
+ * Update menus with new URL.
+ */
+class CRM_Prospect_Upgrader_Steps_Step1008 {
+
+  /**
+   * Runs the upgrader changes.
+   *
+   * @return bool
+   *   Return value in boolean.
+   */
+  public function apply() {
+    (new ProspectingMenu())->updateCaseCategorySubmenus(
```

Comment:
This method needs to be updated to `resetCaseCategorySubmenuUrl`

Reply:
Done: https://github.com/compucorp/uk.co.compucorp.civicrm.prospect/pull/79/commits/be2010ebfa9a8ad0f1cec535c643ffe94845ac34